repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/person-contact.ftl

```diff
@@ -0,0 +1,76 @@
+<#include "../../include/imports.ftl">
+<@hst.setBundle basename="uk.nhs.hee.web.contact,uk.nhs.hee.web.global"/>
+
+<#macro personContact person isAuthor=false>
+    <div class="nhsuk-contact__content">
+        <#if person.image??>
+            <@hst.link var="personImage" hippobean=person.image/>
+            <div class="nhsuk-contact__img-container">
+                <img class="nhsuk-contact__img" src="${personImage}" alt="${person.image.description!}">
```

Comment:
Image isn't centred.
![image](https://user-images.githubusercontent.com/85481555/151801217-9e07f450-3db9-4906-8184-7bbde9676e9c.png)

Reply:
Is there CSS to be added to the build ? I recall Adriana saying that there was a depndency there somewhere